pkgs/tools/admin/lxd/default.nix

```
@@ -98,6 +98,7 @@ buildGoModule rec {
 
   passthru.tests.lxd = nixosTests.lxd;
   passthru.tests.lxd-nftables = nixosTests.lxd-nftables;
+  passthru.updateScript = ./update.sh;
```

I think we can use `gitUpdater`
```suggestion
  passthru.updateScript = gitUpdater {
    url = "https://github.com/lxc/lxd.git";
    rev-prefix = "lxd-";
  };
```

---

Yes, this does work. How can I even know such an updater exists? It's not in the manual.

---

Unfortunately, I don't have an answer. I just saw it being used by other packages.

---

Well thanks for sharing. Better than my shell script for sure. :)